modules/ringieraxelspringerBidAdapter.js

```diff
@@ -267,19 +267,22 @@ const getSlots = (bidRequests) => {
   for (let i = 0; i < batchSize; i++) {
     const adunit = bidRequests[i];
     const slotSequence = deepAccess(adunit, 'params.slotSequence');
-    const creFormat = getAdUnitCreFormat(adunit);
-    const sizes = creFormat === 'native' ? 'fluid' : parseSizesInput(getAdUnitSizes(adunit)).join(',');
+    const creFormats = getAdUnitCreFormats(adunit);
+    const sizes = parseSizesInput(getAdUnitSizes(adunit))?.filter(Boolean);
+
+    if (creFormats.includes('native') && sizes?.indexOf('fluid') === -1) {
+      sizes.push('fluid');
+    }
 
     queryString += `&slot${i}=${encodeURIComponent(adunit.params.slot)}&id${i}=${encodeURIComponent(adunit.bidId)}&composition${i}=CHILD`;
 
-    if (creFormat === 'native') {
-      queryString += `&cre_format${i}=native`;
-    }
+    queryString += `&cre_format${i}=${encodeURIComponent(creFormats.join())}`;
 
-    queryString += `&kvhb_format${i}=${creFormat === 'native' ? 'native' : 'banner'}`;
+    // change 'html' format to 'banner'
+    queryString += `&kvhb_format${i}=${encodeURIComponent(creFormats.map(format => format === 'html' ? 'banner' : format).join())}`;
```

Comment:
a po co to jest? czemu w `getAdUnitCreFormats` nie możemy od razu dodawać `banner` zamiast `html`?

Reply:
`html` idzie do CSRa, `banner` idzie jako kv do GAMa. Ja tylko zostawiam oryginalne nazewnictwo.